src/uid2Sdk.ts

```diff
@@ -81,6 +81,7 @@ export class UID2 {
     return this.getIdentity()?.advertising_token ?? undefined;
   }
   public setIdentity(identity: Uid2Identity) {
+    this.abort(`Identity is set by setIdentity`);
```

Comment:
I think the `abort` call is more than we need. We just need to cancel any in-progress requests.

Reply:
Thanks, I agree, we should just abort all active requests but not reject pending promise. Just updated PR